extensions/rich_text_components/Collapsible/Collapsible.html

```diff
@@ -1,4 +1,10 @@
 <script type="text/ng-template" id="richTextComponent/Collapsible">
+  <style>
+    .conversation-skin-tutor-card-top-content.protractor-test-conversation-content {
+      width: 100% !important;
```

Comment:
We generally try not to use !important, because this is a sort of "override" of the standard CSS priority order, and if a lot of individual things use overrides, then it becomes really hard to figure out what rules apply to what. Would it be possible to find a way to do this that does not require the use of !important?